charts/audit-service/templates/deployment.yaml

```diff
@@ -35,40 +35,51 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
       containers:
-        - name: {{ .Chart.Name }}
-          image: "{{ .Values.global.repository }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
-          ports:
-            - name: web
-              containerPort: 8089
-              protocol: TCP
-          livenessProbe:
-            httpGet:
-              path: /actuator/health
-              port: 8089
-            initialDelaySeconds: 10
-            failureThreshold: 5
-            periodSeconds: 20
-            timeoutSeconds: 5
-          readinessProbe:
-            httpGet:
-              path: /actuator/health
-              port: 8089
-            initialDelaySeconds: 10
-            failureThreshold: 5
-            periodSeconds: 20
-            timeoutSeconds: 5
-          resources:
-          {{- toYaml .Values.resources | nindent 12 }}
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.global.repository }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        ports:
+          - name: web
+            containerPort: 8089
+            protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /actuator/health
+            port: 8089
+          initialDelaySeconds: 10
+          failureThreshold: 5
+          periodSeconds: 20
+          timeoutSeconds: 5
+        readinessProbe:
+          httpGet:
+            path: /actuator/health
+            port: 8089
+          initialDelaySeconds: 10
+          failureThreshold: 5
+          periodSeconds: 20
+          timeoutSeconds: 5
+        resources:
+          {{- toYaml .Values.resources | nindent 10 }}
+        env:
+        - name: "LOADER_PATH"
+          value: {{ .Values.global.persistence.classpathJars.mountPath }}/{{ .Chart.Name }}/classpath/{{ .Chart.Version }}
```

Comment:
```suggestion
          value: {{ required "A valid classPathJars.mounthPath is required" .Values.global.persistence.classpathJars.mountPath }}/{{ .Chart.Name }}/classpath/{{ .Chart.Version }}
```

Reply:
I don't really like the use of {{ required }} since mostly everything in the values.yaml is "required" so far... What value do we get from it here that would be justifiably different from also adding it to all other uses of `{{ .Values.something.else }}` ?

Reply:
When I ran the install raw i got an error relates to classpathJars not being set. I take your point that the problem exists elsewhere - as a compromise I suggest those items that have a valid value set in the values.yaml can be ignored, but those that require the user to configure and set something to have a "required" ??

Reply:
Agreed